src/esm_runscripts/backup/sadfile.py

```diff
@@ -1,6 +1,6 @@
-class sadfile:
+class runfile:
```

Comment:
Not for now, but we should stick to common standards for class names. Normally they are capitalised. One more for the cleanup list.

Reply:
Ah, I missed that. Here is a good read from PEP
https://www.python.org/dev/peps/pep-0008/#class-names

CamelCase is the naming convention for the classes. I remember that some classes in `esm_master` also violate that. Not too bad but the style police might be angry.